src/scripts/oauth/http-refresh-token.service.js

```diff
@@ -8,22 +8,15 @@ function HttpRefreshTokenService($rootScope, $q, $injector, OAuthService, OAuthT
 
   const service = this;
   let cache = null;
-  const limiters = [
-    /www\.bungie.net\/Platform\/User\/*/,
-    /www\.bungie.net\/Platform\/Destiny\/*/
-  ];
+  const matcher = /www\.bungie.net\/Platform\/(User|Destiny)\//;
```

Comment:
You're right, the dot needs escaping. The star before matched zero or more slashes, which didn't seem useful. I think case sensitivity is all right, and yeah we want there to be a slash after User or Platform.